api/types.go

```diff
@@ -239,6 +239,7 @@ type ID struct {
 	RPCProtocolVersion    protocol.ID
 	Error                 string
 	IPFS                  IPFSID
+	Hostname              string
```

Comment:
Let's not name this `Hostname` because it is definitely not a hostname in the DNS-sense of it (it does not matter if it's resolvable and the users can set it to whatever arbitrary value they want). Let's name it `Alias` or simple `Name` (I'm undecided, you, @ZenGround0 ?)

Reply:
Either SGTM

Reply:
`Peername` ?

Reply:
Peername sounds good to me so I've changed it to that. Easy enough to change it if we want it to be something else.